src/stm.py

```diff
@@ -127,31 +108,49 @@ def _cbf_welch(volume, fps, max_freq = 20):
         Frames per second (capture framerate) of the video.
     max_freq : float
         Maximum allowed frequency; frequencies above this are clamped.
-
     Returns
     -------
     retval : array, shape (H, W)
         Heatmap of dominant frequencies at each spatial location (pixel).
     """
     N = nextpow2(volume.shape[0])
-    f, Pxx = signal.welch(volume, fs = fps, window = "hann", nfft = N,
+    f,Pxx = signal.welch(volume, fs = fps, window = "hann", nfft = N,
         return_onesided = True, axis = 0)
-    max_freq_indices = Pxx.argmax(axis = 0)
-    heatmap = f[max_freq_indices]
-    heatmap[heatmap > max_freq] = max_freq
-    return heatmap
+    return get_best_volumes(Pxx,f,max_freq,sequence)
 
 # Utilities
 
+def _get_best_volumes(freq_bins, f, max_freq = 20, n_maps = 5):
+    """
+    The Utility function gets the best maps the top 20 maps from the function
+    Parameters
+    ----------
+    freq_bins : array, shape (F, H, W)
+        The volume of maps from which the infromation has to be selected
+    max_freq : int
+        Maximum allowed frequency; frequencies above this are clamped.
+    n_maps : float
+        top n maps to be selected for heatmaps
+    Returns
```

Comment:
Add a space between the end of the parameters description and the `Returns` header.